addons/MountMuzzle/README.md

```diff
@@ -0,0 +1,155 @@
+**Author:** [Sjshovan (Apogee)](https://github.com/Ap0gee)  
+**Version:** v0.9.2  
+
+
+# Mount Muzzle
+
+> A Windower 4 addon that allows the user to change or remove the default mount music in Final Fantasy 11 Online.
+
+
+### Table of Contents
+
+- [Prerequisites](#prerequisites)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Commands](#commands)
+- [Support](#support)
+- [Change Log](#change-log)
+- [Known Issues](#known-issues)
+- [TODOs](#todos)
+- [License](#license)
+
+___
+### Prerequisites
+1. [Final Fantasy 11 Online](http://www.playonline.com/ff11us/index.shtml)
+2. [Windower 4](http://windower.net/)
+
+___
+### Installation
+1. Navigate to <https://github.com/Ap0gee/MountMuzzle>.
+2. Click on `Releases`. 
+3. Click on the `Source code (zip)` link within the latest release to download.
+4. Extract the zipped folder to `Windower4/addons/`.
+5. Rename the folder to remove the version tag (`-v0.9.2`). The folder should be named `MountMuzzle`.
+ 
+___
+### Usage
+
+**Manually load** the addon by using one of the following commands:
+    
+    //lua load mountmuzzle  
+    //lua l mountmuzzle
+    
+**Automatically load** this addon by adding `lua load mountmuzzle` or `lua l mountmuzzle` to the bottom of your `Windower4/scripts/init.txt` file.
+___    
+### Commands
+
+**help**
+
+Displays available Mount Muzzle commands. Below are the equivalent ways of calling the command:
+
+    //mountmuzzle help
+    //muzzle help
+    //mm help
+    //mountmuzzle h
+    //muzzle h
+    //mm h
+
+**list** 
+
+Displays the available muzzle types. Below are the equivalent ways of calling the command:
+
+    //mountmuzzle list
+    //muzzle list
+    //mm list
+   
+**set _\<muzzle>_**
+
+Sets the current muzzle to the given muzzle type. This command takes a single argument represented by `<muzzle>`. Below are the equivalent ways of calling the command:
+
+    //mountmuzzle set <muzzle>
+    //muzzle set <muzzle>
+    //mm set <muzzle>
+    
+Here are some usage examples for the **set _\<muzzle>_** command: `mm set silent` and `muzzle set zone` etc...
+
+**get**
+
+Displays the current muzzle that is set. Below are the equivalent ways of calling the command:
+	
+    //mountmuzzle get
+    //muzzle get
+    //mm get
+    
+**default**
+
+Sets the current muzzle to the default muzzle type: `Silent`. Below are the equivalent ways of calling the command:
+
+	//mountmuzzle default
+    //muzzle default
+    //mm default
+
+**reload**
+
+Reloads the Mount Muzzle addon. Below are the equivalent ways of calling the command:
+	
+    //mountmuzzle reload
+    //muzzle reload
+    //mm reload
+
+___
+### Support
+**Having Issues with this addon?**
+* Please let me know [here.](https://github.com/Ap0gee/MountMuzzle/issues/new)
```

Comment:
It bothers me more than it should that the period is part of the link :<

Reply:
Done.